File: MoMoPay-iOS-SDK/MoMoSDK/API/MoMoPaySDK.m
//
//  MoMoPaySDK.m
//  SampleApp-Xcode
//
//  Created by Luu Lanh on 9/30/15.
//  Copyright (c) 2015 LuuLanh. All rights reserved.
//

#import "MoMoPaySDK.h"
@implementation MoMoPaySDK
+(void)initializingAppBundleId:(NSString*)bundleid merchantCode:(NSString*)merchantcode merchantName:(NSString*)merchantname merchantNameLabel:(NSString*)merchantnamelabel merchantPublicKey:(NSString*)publickey{
    [MoMoConfig setAppBundleId:bundleid];
    [MoMoConfig setMerchantcode:merchantcode];
    [MoMoConfig setMerchantname:merchantname];
    [MoMoConfig setMerchantnameLabel:merchantnamelabel];
    [MoMoConfig setPublickey:publickey];
    NSLog(@"<MoMoPay> initializing successful");
}
@end

File: SampleApp-Swift/SampleApp-Swift/MoMoAPI/MoMoPaySDK.swift
//
//  MoMoPaySDK.swift
//  SampleApp-Swift
//
//  Created by Luu Lanh on 4/24/17.
//  Copyright © 2017 LuuLanh. All rights reserved.
//

import Foundation

class MoMoPaySDK: NSObject {
    open class func initializingAppBundleId(bundleid: String, merchantCode merchantcode: String, merchantName merchantname: String, merchantNameLabel merchantnamelabel: String, merchantPublicKey publickey: String) {
        
        MoMoConfig .setAppBundleId(bundleId: bundleid)
        MoMoConfig .setMerchantcode(merchantCode: merchantcode)
        MoMoConfig .setMerchantname(merchantName: merchantname)
        MoMoConfig .setMerchantnameLabel(merchantnameLabel: merchantnamelabel)
        MoMoConfig .setPublickey(merchantpublickey:publickey)
        print("<MoMoPay> initializing successful - Your app bundleId \(bundleid)")
    }
}
